synapse/models/science.py

```diff
@@ -0,0 +1,92 @@
+import synapse.lib.module as s_module
+
+class ScienceModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('sci', {
+            'types': (
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', ('guid', {}), {
+                    'doc': 'A hypothesis or theory.'}),
+
+                # TODO link experiment to eventual procedure node
+                ('sci:experiment:type:taxonomy', {}, {}),
+                ('sci:experiment', ('guid', {}), {
+                    'doc': 'An instance of running an experiment to test a hypothesis.'}),
+
+                ('sci:observation', ('guid', {}), {
+                    'doc': 'An observation which may have resulted from an experiment.'}),
+
+                ('sci:evidence', ('guid', {}), {
+                    'doc': 'An assessment of how an observation supports or refutes a hypothesis.'}),
+            ),
+
+            'edges': (
+                (('sci:experiment', 'uses', None), {
+                    'doc': 'The experiment used the target nodes when it was run.'}),
+                (('sci:observation', 'has', None), {
+                    'doc': 'The observations are summarized from the target nodes.'}),
+                (('sci:evidence', 'has', None), {
+                    'doc': 'The evidence includes observations from the target nodes.'}),
+            ),
+
+            'forms': (
+                # TODO many of these forms need author/contact props
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', {}, (
+
+                    ('name': ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'The name of the hypothesis.'}),
+
+                    ('type', ('sci:hypothesis:type:taxonomy', {}), {
+                        'doc': 'The type of hypothesis as a user defined taxonomy.'}),
+
+                    ('summary': ('str', {}), {
+                        'doc': 'A summary of the hypothesis.'}),
+                )),
+
+                # TODO eventually link to a procedure form
+                ('sci:experiment', {}, (
```

Comment:
should this have a :hypothesis property to associate the experiment with a hypothesis?

Reply:
An experiment may produce observations which result in evidence that supports multiple hypotheses. I left this off for now because it may end up requiring a light edge or an array prop.